pkgs/development/python-modules/easysnmp/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, lib
+, pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, python
+, pytest
+, pytestcov
+, pytest-flake8
+, pytest-sugar
+, termcolor
+, ipython
+}:
+
+buildPythonPackage rec {
+  pname = "easysnmp";
+  version = "0.2.5";
+
+  # See https://github.com/kamakazikamikaze/easysnmp/issues/108
+  disabled = !(pythonOlder "3.7");
+
+  src = fetchFromGitHub {
+    owner = "kamakazikamikaze";
+    repo = pname;
+    rev = version;
+    sha256 = "1si9iyxqj6z22jzn6m93lwpinsqn20lix2py3jm3g3fmwawkd735";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    pytest-flake8
+    pytest-sugar
+    termcolor
+    pkgs.net_snmp
+  ];
+
+  buildInputs = [
+    pkgs.net_snmp
+    pkgs.openssl
+    pkgs.gcc
+    ipython
+  ];
+
+  patches = [./easysnmp.patch];
```

Comment:
since this patch is largely trying to fix tests, but the tests fail anyway, you should be able to remove

Reply:
Shouldn't I leave it for posterity in case someone else wants to try to get tests to pass?

Reply:
maybe, but it looks like the current state was for an old version of pytest.... I would say it has bitrotted beyond intended use. 

The most proper thing would be to try and upstream the changes and just amend the next bump. But the repo seems like it's kind of inactive.